simulator/README.md

```diff
@@ -0,0 +1,19 @@
+# Graph Simulator
+
+This is a stand-alone utility that allows us to run different scenarios on the whole of social graph and test
+the desired behavior against the sdk latest implementation.
+
+## Initialized State
+Calculating the initial state for all te graphs and keys is time-consuming, and we've already calculated
+the state for 20000 users in the graph which is uploaded [here](https://drive.google.com/file/d/1WoQ_INPa4aOy4dCm1WUN9KXxq08P0-Rp/view?usp=sharing)
+and if copied inside repository's root folder it can skip the initialization process, and you can directly run
+test scenarios on already established state.
+
+### Build
+For being able to run the 20k users example the **maxPageId** on _frequency.json_ file should be changed to at least **64**.
+We will update this value after it was updated un Frequency chain first.
```

Comment:
based on what is mentioned in the Readme you should only update `frequency.json`. I will add that you will need ot build after so that it rans the latest version

Reply:
@JoeCap08055 If we would like to test the any private friendship scenarios we need at least 64 but since we don't have any use-cases on-chain for it it doesn't make sense to set the chain value to 64 for now.